doc/dependency_decisions.yml

```diff
@@ -217,3 +217,11 @@
     :why: 
     :versions: []
     :when: 2023-01-30 21:31:27.476042000 Z
+- - :license
+  - github.com/yosuke-furukawa/json5
+  - Simplified BSD
+  - :who: sagie
```

Comment:
`who` is empty everywhere else 🤔

Reply:
It is but I used this tool before and you're supposed to run this with `--who` to log who made the "decision" 🤷